src/test/java/uk/gov/ons/census/casesvc/logging/EventLoggerTest.java

```diff
@@ -159,27 +161,23 @@ public void testLogReceiptEvent() {
   @Test
   public void testLogRefusalEvent() {
     // Given
-    RefusalDTO expectedRefusal =
-        convertJsonToRefusalDTO(convertObjectToJson(easyRandom.nextObject(RefusalDTO.class)));
+    RefusalDTO expectedRefusal = easyRandom.nextObject(RefusalDTO.class);
+    expectedRefusal.getCollectionCase().setId(TEST_CASE_ID.toString());
 
     // When
     underTest.logRefusalEvent(
-        new UacQidLink(),
-        "TEST_LOGGED_EVENT",
-        EventType.UAC_UPDATED,
-        expectedRefusal,
-        new EventDTO(),
-        OffsetDateTime.now());
+        new Case(), "TEST_LOGGED_EVENT", EventType.UAC_UPDATED, expectedRefusal, new EventDTO());
```

Comment:
Shouldn't this be `EventType.REFUSAL_RECEIVED?`

Reply:
done